indi-ahp-interferometer/indi_interferometer.xml.cmake

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<driversList>
+<devGroup group="CCDs">
```

Comment:
Should they get their own group? we have receivers IIRC.

Reply:
This code refers to https://github.com/iliaplatone/ahp-interferometer
There is both the chance to use it with receivers (ttl antennas) or photomultiplier/APDs. 
Since the output is a picture it's group should be CCDs.